npm-packages/meteor-installer/README.md

```diff
@@ -1,16 +1,54 @@
 ## Meteor Installer
 
-Node.js <=14.x and npm <=6.x is recommended.
+### Recommended Versions
 
-Install Meteor by running:
+- For Meteor 2
+  - Use Node.js 14.x or higher
+  - Use npm 6.x or higher
```

Comment:
```suggestion
- For Meteor 2
  - Use Node.js 14.x
  - Use npm 6.x
```

For Meteor 2, Node.js and npm should not be higher, right? We  should recommend using Node.js 14 and npm 6 only.

Reply:
Oops, in this case it should be lesser or equal